app/models/manageiq/providers/redfish/inventory/collector/physical_infra_manager.rb

```diff
@@ -1,19 +1,40 @@
 module ManageIQ::Providers::Redfish
   class Inventory::Collector::PhysicalInfraManager < Inventory::Collector
+    def collect
```

Comment:
Move collection of inventory up so that we can perform all of the collection with the RedFish Client session and disconnect then parse later